src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -43,7 +43,7 @@ public class TestConnectionCommand : PSCmdlet, IDisposable
 
         // Count of pings sent to each trace route hop. Default mimics Windows' defaults.
         // If this value changes, we need to update 'ConsoleTraceRouteReply' resource string.
-        private const int DefaultTraceRoutePingCount = 3;
```

Comment:
We should fi `var discoveryAttempts = 0;` too.

Reply:
Maybe in a separate clean-up PR.

Reply:
This gives me the idea to run [RCS1008: Use explicit type instead of 'var' (when the type is not obvious)](https://github.com/JosefPihrt/Roslynator/blob/master/docs/analyzers/RCS1008.md) code diagnostic.

Reply:
There is Roslyn analog too I guess.